wikimedia/wikipedia-ios
# [ALT TEXT] Push commons web view after tapping file name

**Phabricator:** https://phabricator.wikimedia.org/T370332

### Notes
This PR pushes on the commons web view when tapping the file name in the alt text input view. It also fixes a presentation bug where upon tapping the info button in the gallery view, the alt text half sheet modal would not reappear when coming back from the commons web view.

### Test Steps
1. Go through alt-text flow, confirm commons web view appears when tapping the file name.
